docker-compose.yml

```diff
@@ -34,3 +44,11 @@ elasticsearch:
 
 redis:
   image: redis
+
+rabbitmq:
+  image: rabbitmq:3-management
+  hostname: olympia
+  name: olympia
```

Comment:
It seems slightly strange to use `olympia` for the hostname of the rabbitmq server, rather than the `web` container. What does `name` do? I don't see it in any docs.